.gitignore

```diff
@@ -65,6 +65,5 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
-.editorconfig
```

Comment:
May I ask why remove this?

Reply:
I think `.editorconfig` should be tracked by git.